modules/calc/src/main/java/com/opengamma/strata/calc/runner/CalculationParameter.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (C) 2016 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.strata.calc.runner;
+
+import com.opengamma.strata.basics.CalculationTarget;
+import com.opengamma.strata.calc.config.Measure;
+
+/**
+ * The base interface for calculation parameters.
+ * <p>
+ * Parameters are used to control the calculation.
+ */
+public interface CalculationParameter {
```

Comment:
Given how important these are it would be good to go into more detail and include examples of how they are expected to be used.